### ISSUE TITLE
shows black screen after native splash

### ISSUE THREAD

**Issue by ArslanKhan99** [prongbang/screen_protector#12]
shows black screen after native splash

**Comment by PcolBP** [comment]
Yep, it occurs for me as well. While going to app selection view I do see the splash screen, but if I will go to the home screen and then will try to pick up an app from selection view then this time app has blank screen until I'll select it 🤔 

Looks like after minimizing the application splash screen is lost 🤔 Wonder if it is related with flutter or plugin implementation
